rpm/centos-8/Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN \
     yum-builddep --define '_without_btrfs 1' -y /root/rpmbuild/SPECS/*.spec; \
   else \
     yum-builddep --define '_without_btrfs 1' -y /root/rpmbuild/SPECS/docker-c*.spec; \
+    yum-builddep --define '_without_btrfs 1' -y /root/rpmbuild/SPECS/docker-b*.spec; \
```

Comment:
We probably don't need this line; I think this was specifically using `docker-c` so that we only defined this for the `docker-ce-` packages (the engine uses the `btrfs` build-tag); probably the existing one should even be changed to `docker-ce.spec`, as `docker-ce-cli.spec` likely doesn't need it either, but will need to check what we need there.

Reply:
Yes let's do that in a follow-up.